File: app/src/main/java/com/bopr/android/smailer/ui/OptionsActivity.java
package com.bopr.android.smailer.ui;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

/**
 * Options settings activity.
 *
 * @author Boris Pronin (<a href="mailto:boprsoft.dev@gmail.com">boprsoft.dev@gmail.com</a>)
 */
public class OptionsActivity extends AppActivity {

    @NonNull
    @Override
    protected Fragment createFragment() {
        return new OptionsFragment();
    }

}

File: app/src/main/kotlin/com/bopr/android/smailer/ui/OptionsActivity.kt
package com.bopr.android.smailer.ui

import androidx.fragment.app.Fragment

/**
 * Options activity.
 *
 * @author Boris Pronin ([boprsoft.dev@gmail.com](mailto:boprsoft.dev@gmail.com))
 */
class OptionsActivity : BaseAppActivity() {

    override fun createFragment(): Fragment {
        return OptionsFragment()
    }
}